deployments/mainnet/ConvexStakingWrapperAbra.json

```diff
@@ -0,0 +1,1238 @@
+{
+  "address": "0x2F1Ba281cA9c32BCe9C6E8792Ba6448434178f89",
```

Comment:
What wrapper did we end up using and where is the source code for this one?
Convex released an update in between?
This address is diverging from the one used in the cauldron deployment?

Reply:
yes I need to remove this file since it doesn't reflect the current state of deployment.  I first deployed it from what we used then I learn that Convex made a fix so I create a new instance using their factory (clone proxy).

The right address now is 0xb24BE15aB68DC8bC5CC62183Af1eBE9Ecd043250.

I'm removing this file.